tests/conftest.py

```diff
@@ -2353,6 +2353,25 @@ def raredisease_nexflow_config_file_path(raredisease_dir, raredisease_case_id) -
     ).with_suffix(FileExtensions.JSON)
 
 
+@pytest.fixture(scope="function")
+def raredisease_deliverable_data(
+    raredisease_dir: Path, raredisease_case_id: str, sample_id: str
+) -> dict:
+    return {
+        "files": [
+            {
+                "path": f"{raredisease_dir}/{raredisease_case_id}/multiqc/multiqc_report.html",
+                "path_index": "",
+                "step": "report",
+                "tag": ["multiqc-html", "rna"],
```

Comment:
```suggestion
                "tag": ["multiqc-html"],
```

I think it's only "multiqc-html" for DNA cases

Reply:
Added in next commit 👍